.github/workflows/mergify.yml

```diff
@@ -0,0 +1,17 @@
+queue_rules:
+  - name: default
+    conditions:
+      # Conditions to get out of the queue (= merged)
+      - approved-reviews-by=@merbridge/infra-maintainers
+      - approved-reviews-by=@merbridge/control-plane-maintainers
+pull_request_rules:
+  - name: Automatic merge on CI success and review
+    conditions:
+    - or:
+      - "#approved-reviews-by>=1s"
+      - approved-reviews-by=@merbridge/infra-maintainers
```

Comment:
can not read from CODEOWNERS?

Reply:
Yes, mergify can just read from team, besides this, I do not know any optional actions to do this.

Reply:
So if it's a pr from another team, will it work after they approve it?